src/lib/components/pages/anime_info/anime_episodes.svelte

```diff
@@ -92,102 +92,68 @@
-                        >
-                            <div class="flex place-items-center md:gap-[0.7vw]">
-                                <PlayCircle class="w-[1.5vw]" />
-                                <span class="text-[1vw] font-bold leading-none">
-                                    Ep {episode_number}
-                                </span>
-                            </div>
-                        </button>
-                        <div class="flex gap-[0.35vw]">
-                            <button
-                                type="button"
-                                class="btn h-[4vw] w-[4vw] rounded-[0.625vw] bg-surface-900 text-[0.87vw] font-semibold text-surface-50"
-                                disabled
-                            >
-                                <div class="flex flex-col items-center justify-center md:gap-[0.35vw]">
-                                    <Read class="w-[1.3vw]" />
-                                    <div class="flex flex-col items-start">
-                                        <span class="leading-[1vw]">Read</span>
-                                    </div>
-                                </div>
-                            </button>
-                            <button
-                                type="button"
-                                class="btn h-[4vw] w-[4vw] rounded-[0.625vw] bg-surface-900 text-[0.87vw] font-semibold text-surface-50"
-                                disabled
-                            >
-                                <div class="flex flex-col place-items-center md:gap-[0.35vw]">
-                                    <Listen class="w-[1.3vw]" />
-                                    <div class="flex flex-col items-start">
-                                        <span class="leading-[1vw]">Lsiten</span>
-                                    </div>
-                                </div>
-                            </button>
-                        </div>
-                    </show-on-hover>
                 </div>
 
-                <episode-info class="flex h-full w-full flex-col items-start justify-between pt-[0.75vw] md:gap-[0.25vw]">
-                    <div class="flex flex-col items-start md:gap-[0.25vw]">
-                        <span class="font-light text-surface-50/75 transition duration-300 group-hover:text-surface-50 md:text-[0.85vw] md:leading-[1.25vw]">
+                <episode-info class="absolute bottom-0 flex h-auto w-full flex-col items-start justify-between bg-surface-900/75 p-[1vw] transition duration-300 md:gap-[0.75vw] md:backdrop-blur-xl">
+                    <div class="relative flex flex-col items-start md:gap-[0.25vw]">
+                        <span class="w-[18vw] overflow-hidden whitespace-nowrap font-light text-surface-50/90 transition duration-300 group-hover:whitespace-normal group-hover:text-surface-50 md:text-[0.9vw] md:leading-[1.25vw]">
```

Comment:
This is not the right way to handle things. 

Take a look at https://stackoverflow.com/a/55764532